src/main/kotlin/edu/byu/mealplanningassistant/repository/RecipesRepository.kt

```diff
@@ -53,49 +53,37 @@ class RecipesRepository{
         }
     }
 
-    fun getRecipesWithTags(tagSet: HashSet<String>, mealType: String) : List<Recipe>{
-        val filter = `all`("tags", tagSet)
-        return getFromDB(filter)
-    }
+    fun getRecipesWithTags(tagSet: HashSet<String>) = getFromDB(`all`("tags", tagSet))
 
-    fun getRecipes() : List<Recipe> {
-        val filter = exists("_id")
-        return getFromDB(filter)
-    }
+    // TODO I'm not sure if this is supposed to do something different than getAllRecipes later... I don't think it's pertinent to decide this now, but should discuss and consolidate if possible later
```

Comment:
I just had this for testing - I think it's nice to be able to see everything in the DB but we won't actually use it in production -- more just like a backend hack.